R/toolbar.R

```diff
@@ -0,0 +1,34 @@
+#' Toolbar component
+#'
+#' @description
+#' A toolbar which can contain buttons, inputs, and other UI elements in a small
+#' form suitable for inclusion in card headers, footers, and other small places.
+#'
+#' @param ... UI elements for the toolbar.
+#' @param align Determines if toolbar should be aligned to the `"right"` or
+#'   `"left"`.
+#' @param size The size of the toolbar. Must be one of `"sm"`, `"md"`, or
+#'   `"lg"`.
+#'
+#' @return Returns a toolbar element.
+#'
+#' @export
+toolbar <- function(
+  ...,
+  align = c("right", "left"),
+  size = c("sm", "md", "lg")
```

Comment:
I think this is a good idea in the long term and that we'll need a `size` feature down the line. For now though, it's not super clear how `size` on `toolbar()` will propagate to toolbar items.

For now, I think we're better off targeting small input and toolbar sizes and once we have the pieces in place we can tackle `size` on `toolbar()` passing down to its items. Can you make sure we track this somewhere?

Reply:
Will need to keep `font-size` for small settings